### ISSUE TITLE
S U G G E S T I O N S

### ISSUE THREAD

**Issue by FrenzyExists** [FrenzyExists/aquarium-vim#6]
Heyy Sup! Decided to open an issue here which is more of a Suggestions forum, drop any crazy ideas for the theme here!

**Comment by bakageddy** [comment]
Tree sitter support would be awesome! If you plan on porting this scheme to nvim, I can learn and help you create that port. Thanks!

**Comment by bakageddy** [comment]
and if you want to add some features, I would be more than glad to help you out.

**Comment by bakageddy** [comment]
Feature Idea: Add Visual Inverse Selection, like the one from morhetz/gruvbox.

**Comment by FrenzyExists** [comment]
https://github.com/FrenzyExists/aquarium-vim/issues/6#issuecomment-951901655

I haven't heard of the visual inverse selection before, i'll give a look to how gruvbox does this :+1: 

About Treesitter, I think is a good idea, tho, considering how nvim is doing, a Lua version of the aquarium theme would more likely help with this, as Treesitter is in Lua and more and more things in neovim are now being made in lua. I've been working on a nvim/lua version of this theme, tho, still not much. For this repo, maybe a thing or two that are lua focused would be fine, but mostly it should keep itself on vim-made plugins, since hopefully, a lua theme should be around soon

**Comment by bakageddy** [comment]
Then you can merge my pr when you port this theme to lua, Thank you!

**Comment by FrenzyExists** [comment]
No prob :+1:

**Comment by FrenzyExists** [comment]
Aquarium wasn't really the colors you see now, the original aquarium is now added under the name of Mish, he was the guy who started the idea, and made the first colors. Then I kinda continue working on the theme and adjusted colors as time passed, till what you see today

**Comment by darquill** [comment]
Hey, I made the .Xresources file for the color scheme, maybe you could add it under extras so people can use it say with https://terminal.sexy to export to their terminal of choice config file:

```
! special
*.foreground:   #c8cedc
*.background:   #20202a
*.cursorColor:  #b8dceb

! black
*.color0:       #3b3b4d
*.color8:       #1b1b23

! red
*.color1:       #ebb9b9
*.color9:       #d95e59

! green
*.color2:       #caf6bb
*.color10:      #8fc587

! yellow
*.color3:       #E6DFB8
*.color11:      #ffcf85

! blue
*.color4:       #cddbf9
*.color12:      #4a83c3

! magenta
*.color5:       #f6bbe7
*.color13:      #bf83b5

! cyan
*.color6:       #b8dceb
*.color14:      #4eb3cd

! white
*.color7:       #c8cedc
*.color15:      #abb2c2
```

**Comment by FrenzyExists** [comment]
Feel free to make a PR :+1:

**Comment by FrenzyExists** [comment]
@darquill I added the file :+1:

**Comment by darquill** [comment]
Cheers, I saw the prev message, but then I forgot to make the PR. Glad you added it :) Thanks!

**Comment by bakageddy** [comment]
Any Idea on porting this  theme to emacs?

**Comment by FrenzyExists** [comment]
I don't really use emacs, and  don't really have any experience with emacs lisp. If someone knows emacs lisp and wants to port it there, it be cool. The PR could be in my dotfiles, which is aquarium themed

**Comment by bakageddy** [comment]
ok, I will create a pr soon.

**Comment by gmcusaro** [comment]
It's not a crazy idea 😅 but, the support for [nvim-tree](https://github.com/kyazdani42/nvim-tree.lua) (a file explorer written in Lua) would also be useful.

**Comment by FrenzyExists** [comment]
I actually use nvim-tree, haven't tweaked its colors (yet), I think its actually useful indeed

I could prolly add it, won't be this week i think, college stuff, but afterwards i think i could. If anyone went ahead and implement it one can make a PR

**Comment by FrenzyExists** [comment]
@gmcusaro 

News: Aquarium now has NvimTreee support. Give a check in case i forgot something :+1: 


Welp... not sure what else to do, i guess there's nothing else?

**Comment by gmcusaro** [comment]
@FrenzyExists
Great! Everything seems OK with NvimTree!

**Comment by gmcusaro** [comment]
During workflow in a room with suffused light and medium/low brightness of the screen I noticed low contrast between the commented code and the background.

The comment color `#3D4059`, with the background color `#20202A` have a contrast ration of 1.59:1. One suggestion would be to replace comment color with `#44495E` with a better ratio of 1.81:1.

**Comment by FrenzyExists** [comment]
Before:
![image](https://user-images.githubusercontent.com/47986470/144786598-951bb94f-d596-4716-8c00-1af7ec226472.png)

After:
![image](https://user-images.githubusercontent.com/47986470/144786650-54bf7002-bd0c-4c45-8d98-d7e32c87480d.png)

Additional testing with `#4b4e67` or 14% brighter from original color:
![image](https://user-images.githubusercontent.com/47986470/144786904-a0fd1305-e910-455f-8650-8e3a79c58918.png)

**Comment by FrenzyExists** [comment]
I think i'll go with your color, idk why making the original brighter give this feel that it might blend with the default foreground, yours have a nice contrast that is easy to make what is a default, and a comment

**Comment by FrenzyExists** [comment]
![image](https://user-images.githubusercontent.com/47986470/144787544-54819aa0-9a7c-4243-886a-8af8175d6c01.png)

Not gonna lie, it makes it much easier to read. Gonna add the changes :+1: 

Edit: Done!

**Comment by gmcusaro** [comment]
@FrenzyExists yeah, too light wouldn't do. It is lighter and a little more gray than blue. Well done!

**Comment by jiyometrik** [comment]
support for more neovim-only plugins:
- barbar
- bufferline.nvim
- telescope 
- lspsaga

**Comment by FrenzyExists** [comment]
First time I hear of lspsaga. Gonna give a look at em and see what I can do. If anyone wants to get ahead and add support feel free to PR :+1:

**Comment by oblitum** [comment]
I don't like undercurls for links... Always used them for errors and spell mistakes, it feels really weird for HTML/Markdown links. Links almost always are underline, so, I wished that wasn't hardcoded.

**Comment by nayaabkhan** [comment]
@FrenzyExists Any idea how to make `cursorline` work? I have enabled it but the current line is not highlighted when using this color scheme.

**Comment by rhyses-pieces** [comment]
This looks beautiful! Is it okay to port this theme to Wezterm?

**Comment by FrenzyExists** [comment]
Sure! Feel free to do so.

**Comment by pr3c0g** [comment]
Hi there! I would really love to get that wallpaper shown in the [terminal screenshot](https://github.com/FrenzyExists/aquarium-vim/blob/develop/screenshots/preview-dark.png) :) . Since this is a suggestions thread, I suggest you add a link to it in the readme, if that's ok.
